fhir-config/src/main/java/com/ibm/fhir/config/FHIRConfiguration.java

```diff
@@ -135,6 +142,7 @@ public class FHIRConfiguration {
     public static final String DEFAULT_TENANT_ID_HEADER_NAME = "X-FHIR-TENANT-ID";
     public static final String DEFAULT_DATASTORE_ID_HEADER_NAME = "X-FHIR-DSID";
     public static final String DEFAULT_PRETTY_RESPONSE_HEADER_NAME = "X-FHIR-FORMATTED";
+    public static final String DEFAULT_SHARD_KEY_HEADER_NAME = "X-FHIR-SHARD-KEY";
```

Comment:
this one should go into both https://ibm.github.io/FHIR/Conformance#http-headers and https://ibm.github.io/FHIR/guides/FHIRServerUsersGuide#54-custom-http-headers when we're ready to advertise it (#3698)

Reply:
I'll create a separate issue for this: #3714